bedrock/firefox/templates/firefox/privacy/book.html

```diff
@@ -87,11 +87,11 @@ <h3>{{ ftl('privacy-book-the-web-never') }}</h3>
     aspect_ratio='mzp-has-aspect-1-1',
     class='mzp-l-card-feature-right-half mzp-t-firefox'
   ) %}
-    <h3>{{ ftl('privacy-book-trackers-everywhere-') }}</h3>
+    <h3 id="trackers-everywhere">{{ ftl('privacy-book-trackers-everywhere-') }}</h3>
```

Comment:
There is another link at the bottom of this page that is just `trackers`. Not sure if that's confusing.